src/common/mmo.h

```diff
@@ -256,6 +256,12 @@
 #define MAX_ELESKILLTREE 3
 #endif
 
+// Maximum item options [Smokexyz]
+#ifndef MAX_ITEM_OPTIONS
+#define MAX_ITEM_OPTIONS 5
+STATIC_ASSERT(MAX_ITEM_OPTIONS <= 5, "This value is limited by the client and database layout and should only be increased if you know the consequences.");
```

Comment:
Can you move this line to after the `#endif`? (so that if it's defined elsewhere it still asserts)